lib/nbv.js

```diff
@@ -71,6 +71,19 @@ var nbv = (function() {
 
         // if (st.toc)
         //     render_toc()
+
+        // fixed: postprocessing of tex by nagexiucai
+        //        must do below after [marked](https://github.com/chjj/marked)
+        //        for exsample: render tex by '[kates](https://github.com/Khan/KaTeX)
+        // var maths = document.getElementsByClassName("math");
+        // for(var i=0; i<maths.length; i++) {
+        //   katex.render(maths[i].getAttribute("tex"), maths[i], {
```

Comment:
If you'll be using this code to implement KaTeX, please use single quotes instead of double quotes, just for consistency sake.